universe/config.json

```diff
@@ -42,14 +42,23 @@
                     "description": "The location of a volume on the host to be used for persisting Jenkins configuration and build data. The final location will be derived from this value plus the name set in `name` (e.g. `/mnt/host_volume/jenkins`). Note that this path must be the same on all DC/OS agents.",
                     "type": "string"
                 },
-                "persistent-volume-size": {
+                "local-persistent-volume-size": {
```

Comment:
This is a breaking "API" change.

Leave it as is. I don't see a reason to have a separate field for the two.

I would actually make external a boolean flag, and then consume the name if the boolean is true.